tests/test_client.py

```diff
@@ -83,6 +83,12 @@ def test_index_list_with_params(client, user):
     assert data_by_ids.status_code == 200
     data_list_all = data_by_ids.json
 
+    # with nonstrict prefix
+    stripped = rec_1["did"].split("testprefix:", 1)[1]
+    data_by_ids = client.get("/index/?ids={}".format(stripped))
+    assert data_by_ids.status_code == 200
+    data_list_all = data_by_ids.json
+
```

Comment:
I guess I would also add different types of test cases like the one we talked about in chat where a guid and a pref+guid might exist. That is something we definitely want to handle.